stories/packages/AlertDialogStories/index.tsx

```diff
@@ -0,0 +1,160 @@
+import React, {ReactElement, useState} from 'react';
+import Modal from 'react-native-modalbox';
+import {TouchableOpacity} from 'react-native';
+
+import AlertDialog from '../../../packages/AlertDialog/index';
+import {ContainerDeco} from '../../../storybook/decorators';
+import {storiesOf} from '@storybook/react-native';
+import styled from '@emotion/native';
+
+const Container = styled.SafeAreaView`
+  flex: 1;
+  align-self: stretch;
+  justify-content: center;
+  align-items: center;
+`;
+
+const ResultText = styled.Text`
+  font-size: 16px;
+  margin-top: 24px;
+`;
+
+const ExampleText = styled.Text`
+  font-size: 20px;
+  color: cornflowerblue;
+  margin-bottom: 15px;
+`;
+
+function Dialog(): React.ReactElement {
+  const [dialogResult, setDialogResult] = useState<string | boolean | null>('');
+
+  const alertRef: React.RefObject<Modal> = React.createRef();
+  const confirmRef: React.RefObject<Modal> = React.createRef();
+  const promptRef: React.RefObject<Modal> = React.createRef();
```

Comment:
Class Component: only `createRef` can be used.
Function Component: Both `createRef` and `useRef` can be used.

But, in function component, `ref` by `createRef` will be initialize at every re-render.
See [impementaion](https://github.com/facebook/react/blob/8ccfce460f141299d61290f877745407e05e531e/packages/react/src/ReactCreateRef.js#L14) of `createRef`.

[`useRef`](https://reactjs.org/docs/hooks-reference.html#useref) persists for the full lifetime of the component.

So `useRef` is better choice here.